### DISCUSSION TITLE
Newsletter section Using Mono theme - confused or just doing things wrong?

### DISCUSSION THREAD

**Discussion by bjazmoore** [GetPublii/Publii#1665] [Help]
So I have installed the Mono theme.  It is really nice.  I want to put my own custom newsletter code and link into the newsletter section.  Here is what I have in the field where the form goes:

I am having trouble pasting in the code - I will add it to the first comment.

Here is what it looks like:

![2](https://github.com/user-attachments/assets/f78c5c6b-bd21-415e-8526-fcd2a191fd36)

I keep getting the default newsletter message above my message:

![1](https://github.com/user-attachments/assets/f1d4cbee-b204-4dda-9c2b-a9cc5f6aaef6)

I am sure I can remove that in the theme file, but I would really like to use the theme in its vanilla form, as it will be for a client.

What am I doing wrong?

**Comment by bjazmoore** [comment]
![3](https://github.com/user-attachments/assets/2ecaef44-9384-4280-860a-4fbe0a92a699)

**Comment by bjazmoore** [comment]
I found the text in mono.lang.json.

![4](https://github.com/user-attachments/assets/2ec02c73-3689-43cd-9969-39b17fa2bc09)

I kind of with these were fields in the theme.   I know I can make that change myself, but then of course the theme will no longer be vanilla and at risk of being overwritten when later updates come out.

I am closing this issue and opening a feature request.

**Reply by candidexmedia** [reply]
To avoid overwrites, try theme overrides:

https://getpublii.com/dev/theme-overrides/
